src/os/unix/pa_unix_hostapis.c

```diff
@@ -99,5 +101,9 @@ PaUtilHostApiInitializer *paHostApiInitializers[] =
         PaSkeleton_Initialize,
 #endif
 
+#if PA_USE_OPENSLES
+        PaOpenSLES_Initialize,
+#endif
```

Comment:
I'm not sure if there are any OpenSL ES implementations for Windows, but if there are, this should be added to pa_win_hostapis.c as well.

Reply:
I had a look around, there [seems to be 1 other implementation around according to wikipedia](https://en.wikipedia.org/wiki/OpenSL_ES#Implementations), and I'm pretty sure the 2nd implementation is not open source :/

Reply:
IMO we should apply [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)

Even if OSLES is supported on Windows I can't imagine it being the better alternative to any of the other available backends

And if another platform is truly benefited from OSLES, then that can be evaluated when the time arrives